storage/Holders/Holder.h

```diff
@@ -57,11 +57,21 @@ namespace storage
 	Device* get_source();
```

Comment:
This header file is sorely missing the explanation what a _holder_ is.
It is simple when you had any graph theory during studying; it's the edges connecting nodes. But this is not clear unless one knows the lingo of the underlying Boost graph library.